DashWallet/Sources/UI/CrowdNode/Portal/Model/CrowdNodeTransferModel.swift

```diff
@@ -107,12 +107,13 @@ final class CrowdNodeTransferModel: SendAmountModel {
         return displayString
     }
 
-    override var isSendAllowed: Bool {
+    override var isAllowedToContinue: Bool {
+        // TODO: Do we need to have a different logic here depends on direction?
```

Comment:
@Syn-McJ I think we need to have a little bit different logic here depending on direction. `super.isAllowedToContinue` also checks whether you have enough amount in your wallet which should not be checked when we withdraw from CrowdNode.

Reply:
@tikhop this is fine I think because the check is performed in the `canShowInsufficientFunds` which is overridden below.

Reply:
@Syn-McJ Got it - makes sense. I will remove the todo.